components/HeaderAppBar/index.tsx

```diff
@@ -32,7 +32,7 @@ export default function HeaderAppBar(props: HeaderAppBarProps) {
   }
 
   return (
-    <AppBar color="transparent" sx={{ px: 2 }}>
+    <AppBar color="transparent" position="relative" sx={{ px: 2 }}>
```

Comment:
this change fixes the display issue that we kind of patched earlier. the content would still appear slightly under the bar but this doesn't happen anymore